addons/slack/install.js

```diff
@@ -13,64 +13,97 @@ const getTemplateFiles = (fileNames) =>
   }, {});
 
 module.exports = async (ctx) => {
-  let storageSpecification = await Sdk.createStorage(ctx);
+  if (ctx.caller.permissions) {
+    // Create the Add-On Handler
+    try {
+      // Capture resource path and remove it as we will not need it in the addon instance
+      const eventsResourcePathKey = 'fusebit_events_resource_path';
+      let eventsResourcePath = ctx.body.configuration[eventsResourcePathKey];
+      delete ctx.body.configuration[eventsResourcePathKey];
 
-  // Create the Add-On Handler
-  try {
-    await Sdk.createFunction(ctx, {
-      configurationSerialized: `# Add-on configuration settings
+      await Sdk.createFunction(
+        ctx,
+        {
+          configurationSerialized: `# Add-on configuration settings
```

Comment:
Is it still necessary to have both the serialized and nonserialized elements, or can the serialized ones be removed entirely?  Yes, the comments will be lost, but they don’t add anything anyways.

Reply:
We only use the serialized across all of the addons, and I don't mind the comment. Won't fix for now.